contents/docs/session-replay/manual.mdx

```diff
@@ -1,5 +1,5 @@
 ---
-title: Product Manual
+title: Session Replay
```

Comment:
Maybe we call with Getting started? Feels weird to have a page in the session replay docs called "session replay"

Reply:
Yeah it is a bit weird, but this is consistent with the other pages like this (e.g. feature flags)